io.catenax.sealant.sealant_pass/1.0.0/metadata.json

```diff
@@ -0,0 +1 @@
+{ "status" : "draft"} 
```

Comment:
adjust to release prior ms3

Reply:
why release? isn't it a draft?? The semantic Hub should show it as draft.